src/codeu/chat/common/NetworkCode.java

```diff
@@ -31,6 +31,7 @@
       NEW_USER_RESPONSE = 12,
       NEW_CONVERSATION_REQUEST = 13,
       NEW_CONVERSATION_RESPONSE = 14,
+      REMOVE_CONVERSATION_REQUEST = 15,
```

Comment:
Is it required there be a response if the user shouldn't be expected anything after making the request? Maybe a message saying if the conversation was created successfully but I don't think that would need a response.